conf/messages.cy

```diff
@@ -94,6 +95,11 @@ cf.cash-account-detail.back-to-cash-account=Yn ôl i ‘Cyfrif arian parod’
 cf.cash-account-detail.exceeded-threshold = Mae gormod o drafodion o’r 6 mis diwethaf i’w harddangos.
```

Comment:
might be worth adding a newline below like you're doing in the .en

Reply:
Done